Resources/views/Collector/db.html.twig

```diff
@@ -257,7 +257,8 @@
         }
 
         function expandAllQueries(button) {
-            var queries = document.getElementsByClassName('query-section'),
+            var conn = button.getAttribute('data-connection'),
+                queries = document.getElementById('queries-' + conn).getElementsByClassName('query-section'),
```

Comment:
FYI, in the Symfony core we use `querySelectorAll()` instead (see also http://tanalin.com/en/blog/2014/03/get-elements-by-class-name-ie8/)